<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mmainwindow</class>
 <widget class="QMainWindow" name="mmainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>671</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="nameofAuto_label">
          <property name="styleSheet">
           <string notr="true">font-size: 24px; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Автошкола Автолюкс</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="maintext_label">
          <property name="styleSheet">
           <string notr="true">font-size: 18px; margin-bottom: 20px;
</string>
          </property>
          <property name="text">
           <string>Главное меню</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="empButton">
            <property name="styleSheet">
             <string notr="true">
             QPushButton {
                 background-color: #4CAF50;
                 color: white;
                 font-size: 16px;
                 padding: 10px;
                 border-radius: 5px;
             }
             QPushButton:hover {
                 background-color: #45a049;
             }
            </string>
            </property>
            <property name="text">
             <string>Форма сотрудников</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="kurButton">
            <property name="styleSheet">
             <string notr="true">
             QPushButton {
                 background-color: #2196F3;
                 color: white;
                 font-size: 16px;
                 padding: 10px;
                 border-radius: 5px;
             }
             QPushButton:hover {
                 background-color: #0b7dda;
             }
            </string>
            </property>
            <property name="text">
             <string>Форма курсантов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="paperButton">
            <property name="styleSheet">
             <string notr="true">
             QPushButton {
                 background-color: #f44336;
                 color: white;
                 font-size: 16px;
                 padding: 10px;
                 border-radius: 5px;
             }
             QPushButton:hover {
                 background-color: #da190b;
             }
            </string>
            </property>
            <property name="text">
             <string>Статьи</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="timetButton">
            <property name="styleSheet">
             <string notr="true">
             QPushButton {
                 background-color: #ff9800;
                 color: white;
                 font-size: 16px;
                 padding: 10px;
                 border-radius: 5px;
             }
             QPushButton:hover {
                 background-color: #e68900;
             }
            </string>
            </property>
            <property name="text">
             <string>Расписание занятий</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exitButton">
            <property name="styleSheet">
             <string notr="true">
             QPushButton {
                 background-color: #9E9E9E;
                 color: white;
                 font-size: 16px;
                 padding: 10px;
                 border-radius: 5px;
             }
             QPushButton:hover {
                 background-color: #757575;
             }
            </string>
            </property>
            <property name="text">
             <string>Выход</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
